### ISSUE TITLE
"Add new site" button disappears below the fold/scroll

### ISSUE THREAD

**Issue by klimeryk** [Automattic/wp-calypso#49655]
### Steps to reproduce the behavior

1. Go to wordpress.com
2. Click on Switch Site.
3. Try to click the Add new site button that should be on the bottom of the screen.

#### What I expected to happen

Normally (when you did not scroll at all since refresh... so very rare, I guess):

<img width="277" alt="Screenshot 2021-02-03 at 09 08 59" src="https://user-images.githubusercontent.com/3392497/106724120-bf91c780-65ff-11eb-83ce-9384e3bedf50.png">

#### What actually happened

As soon as you scroll the main area, the Add new site link disappears below the fold:
<img width="275" alt="Screenshot 2021-02-03 at 09 10 26" src="https://user-images.githubusercontent.com/3392497/106724183-cf111080-65ff-11eb-8a26-7a7f13c7c715.png">

And even if you scroll all the way down the sidebar, it's not visible (notice the weird position of the scrollbar, going "under" the viewport):

<img width="268" alt="Screenshot 2021-02-03 at 09 10 02" src="https://user-images.githubusercontent.com/3392497/106724256-ec45df00-65ff-11eb-8117-7800996c3707.png">

You have to scroll the right/main part of the screen instead - which is completely counterintuitive and unless you _know_ that's where the button should be, it could be very easy to miss.

This is one of the main actions existing users might want to do (and it's not easy to do it any other way). It's already hard to find it normally, let's make sure it's not even harder ;)

**Comment by getdave** [comment]
@klimeryk Thanks for reporting.

I tried but was unable to replicate this.

Please could you retrest and check this is still manifesting? If so then feel free to remove the `Not Reproducable` label and then could you provide the browser and screensize where you found this issue?

Much appreciated.

**Comment by klimeryk** [comment]
@getdave,. thanks for having a look! Yes, I can still reproduce. I've tried originally with Firefox 86.0b6 (Developer Edition), now I've also checked in stable (85.0.1) and Safari (14.0.2) and can trigger it in all.
My screensize might have to do something with it - it's a 13'' MBP (1440x900 Retina).

Happy to jump on a call to show to how I'm reproducing it - the key is to scroll the right (main) part of Calypso first, apparently.
